internal/docstore/dynamodocstore/dynamo.go

```diff
@@ -158,13 +158,27 @@ func (c *collection) get(ctx context.Context, doc driver.Document, fieldpaths []
 	if err != nil {
 		return err
 	}
-	if len(fieldpaths) > 0 {
-		return errors.New("Get with field paths unimplemented")
-	}
+
 	in := &dyn.GetItemInput{
 		TableName: &c.table,
 		Key:       av.M,
 	}
+	if len(fieldpaths) > 0 {
+		// Construct a projection expression for the field paths.
+		// See https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Expressions.ProjectionExpressions.html.
+		nbs := []expression.NameBuilder{expression.Name(docstore.RevisionField)}
+		for _, fp := range fieldpaths {
+			nbs = append(nbs, expression.Name(strings.Join(fp, ".")))
```

Comment:
Forgot about this probably, why do we store a fieldpath as a `[]string`?

Reply:
For the convenience of drivers, so they don't have to parse and validate a dot-separated string.